whatlies/transformers/ivis.py

```diff
@@ -0,0 +1,79 @@
+from whatlies import EmbeddingSet
+from whatlies.transformers.common import embset_to_X, new_embedding_dict
+
+from ivis import Ivis as IVIS
+import numpy as np
+
+
+class Ivis:
+    """
+    This transformer scales all the vectors in an [EmbeddingSet][whatlies.embeddingset.EmbeddingSet]
+    by means of Ivis algorithm. We're using the implementation found in
+    [beringresearch/ivis](https://github.com/beringresearch/ivis)
+
+    Arguments:
+        n_components: the number of compoments to create/add
+        kwargs: keyword arguments passed to the Ivis implementation
+
+    Usage:
+
+    ```python
+    from whatlies.language import GensimLanguage
+    from whatlies.transformers import Ivis
+
+    words = ["prince", "princess", "nurse", "doctor", "banker", "man", "woman",
+             "cousin", "neice", "king", "queen", "dude", "guy", "gal", "fire",
+             "dog", "cat", "mouse", "red", "bluee", "green", "yellow", "water",
+             "person", "family", "brother", "sister"]
+
+    lang = GensimLanguage("wordvectors.kv")
+    emb = lang[words]
+
+    emb.transform(Ivis(3)).plot_interactive_matrix('ivis_0', 'ivis_1', 'ivis_2')
+    ```
+    """
+
+    def __init__(self, n_components=2, **kwargs):
+        self.is_fitted = False
+        self.n_components = n_components
+        self.kwargs = kwargs
+
+    def __call__(self, embset):
+        if not self.is_fitted:
+            self.fill_arguments(embset)
+            self.tfm = IVIS(embedding_dims=self.n_components, **self.kwargs)
+            self.fit(embset)
+        return self.transform(embset)
+
+    def fill_arguments(self, embset):
```

Comment:
This can be a hidden method, the end user won't need to see it and I'd prefer it not show up in the documentation.

Reply:
Come to think of it, do we really need this method? Why not just pass the `**kwargs` directly?